auction-server/src/auction.rs

```diff
@@ -1488,7 +1488,7 @@ async fn run_submission_loop<T: ChainStore>(
                             }
                             models::ChainType::Svm => {
                                 if let Some(chain_store) = store
-                                    .chains
+                                    .chains_svm
```

Comment:
I think this is one of the places where panic is more appropriate. Otherwise, we will discover it in prod :D

Reply:
We should find a way to make it compile time :D

Reply:
`submit_auctions::<ChainStoreEvm>(store.clone(), chain_store, chain_id).await`
:D